Functions/Import/xASL_imp_DCM2NII_ConvertScan.m

```diff
@@ -77,16 +79,21 @@
         fprintf('Subject = %s, visit = %s, session = %s, scan = %s\n',scanFields.subjectID, scanFields.visitID, printSession, scan_name);
     end
 
-    % Defaults
-    bOneScanIsEnough = false;
-    bPutInSessionFolder = true; 
+%     % Defaults
+    bOneScanIsEnough = false; % PM: THIS USED TO BE TRUE FOR ANATOMICAL SCANS
+    bPutInSessionFolder = true; % THIS USED TO BE FALSE FOR ANATOMICAL SCANS
     switch scan_name
         case {'ASL4D', 'M0', 'ASL4D_RevPE', 'func_bold'}
             bPutInSessionFolder = true;
-        case {'T1', 'WMH_SEGM', 'FLAIR', 'T2', 'T1c'}
-            bPutInSessionFolder = false;
+            bAnatomical = false;        
+        case {'T1w' 'T2w' 'T1', 'WMH_SEGM', 'FLAIR', 'T2', 'T1c'}
+%             bPutInSessionFolder = false;
```

Comment:
I don't get this change. bPutInSessionFolder is initialized as true, so if you comment it here, it will put all anat to session folders....

Reply:
You need this, otherwise in some cases, import won't work correctly. NIfTI2BIDS (or just before that) the code assumes that even T1's are in their own folders

Reply:
But previously. This worked rather correctly and you are now changing this behavior. That might be problematic.

Reply:
Well test it, it doesn't really work nicely, because `NII2BIDS` assumes folders for all scantypes, if I remember correctly. Perhaps especially if you have multiple T1 or multiple FLAIR scans. But there was something going wrong.

Reply:
We need to check together then.